site/content/kapp-controller/docs/latest/install.md

```diff
@@ -57,3 +57,9 @@ rules:
 ```
 3. Set the `IMGPKG_ENABLE_IAAS_AUTH` [environment
    variable](https://carvel.dev/imgpkg/docs/latest/auth/#via-iaas) to false.
+
+   ### Kubernetes versions < 1.20
+   Starting in kapp-controller 0.31.0 we have upgraded our underlying kubernetes
```

Comment:
Can we provide more details on the logging issue so users understand why to disable the APIs? Maybe we can also link to to feature gate docs so it's more clear what is being disabled?

Can we also show how to do this through the configuration? Something like kbld -f ... | ytt -v enable_APIPriorityAndFairness=true | kapp deploy -a kc -f-.

Reply:
The only way I tested disabling them was through configuration, but I did it by changed the value in the values.yaml file directly.

To be honest I have some knowledge-gaps around how our installation process works in practice.  My suspicion is that our in-house users have a different flow than an OSS user, and the most generally applicable thing is to say "set this variable to false, I don't really have a stake in how you do that."  But I could be wrong.

Reply:
> Can we provide more details

I think i've done this part